src/sagas/index.js

```diff
@@ -0,0 +1,7 @@
+import watchGetData from '../components/AlertsNotifications/alerts-saga';
```

Comment:
Why is the name different from the file being imported, and why aren't you using camelCase?